daemon/algod/api/algod.oas2.json

```diff
@@ -3972,6 +4045,74 @@
           }
         }
       }
+    },
+    "SimulationUnnamedGroupResources": {
+      "description": "If unnamed resource access is allowed, this is the subset of unnamed resources that were accessed by this group and could benefit from group resource sharing. In other words, there are no restrictions on where these resources must be placed in the transaction group. Also see property 'unnamed-resources-accessed' in SimulateTransactionResult to see unnamed resources accessed by each transaction which cannot benefit from group sharing; in contrast, these resources must be placed local to that transaction.",
+      "type": "object",
+      "required": [
+        "resources"
```

Comment:
Why is `resources` required?  You might need to fill this with some asset-holdings and or app-locals", but no top-level resources are missing..

Reply:
Perhaps we should tuck asset-holdings and `app-locals` into the `SimulationUnnamedResourceAssignment`? In the single transaction context, I suppose they could be blank. Or, perhaps it is actually a little useful to see which cross-product refs you accessed (even though the problem is sure to be fixed simply by adding the main resources).

Reply:
I consolidated the two structs into just `SimulateUnnamedResourcesAccessed ` in 9b112ee23.

For the non-resource sharing case, the `logic.UnnamedResourcePolicy` never gets invoked for cross product access, so it would be difficult to report anything close to accurate there. So I'm fine with unnamed cross products remaining blank in the response, since they wouldn't be useful in assigning resources anyway.